lib/util/src/bloom_filter.c

```diff
@@ -1,13 +1,16 @@
 /* SPDX-License-Identifier: Apache-2.0 */
 /*
- * Copyright (C) 2015-2020,2022 Micron Technology, Inc.  All rights reserved.
+ * Copyright (C) 2015-2022 Micron Technology, Inc.  All rights reserved.
  */
 
-#include <hse_util/bitmap.h>
+#include <bsd/bitstring.h>
+
 #include <hse_util/bloom_filter.h>
 #include <hse_util/logging.h>
 #include <hse_util/page.h>
 
+#define BYTE_SHIFT 3
```

Comment:
where is this used?  is there a more descriptive name?

Reply:
That is the name that came from the original file. It is used in that file.

Reply:
We can just use `* CHAR_BIT` instead if that is clearer.

Reply:
What you don't see is that I pared down bitmap.h a long long time ago.  It used to be this enormous monstrosity with many many functions to support various words size, complete with it's own unit test!

Reply:
No, CHAR_BIT is 8, so you'd have to turn all the shifts into multiplies and divides which will run really really slow on -O0 builds.  Already root spill is extremely slow on -O0 builds, so changing this could potentially make debug builds useless...
But I thought you eliminated BYTE_SHIFT via the bit_* stuff?

Reply:
Yes to the multiply/divide if changing to CHAR_BIT. Just didn't want to type that out in the previous message.